src/test/rustdoc-gui/toggle-docs-mobile.goml

```diff
@@ -0,0 +1,7 @@
+goto: file://|DOC_PATH|/test_docs/struct.Foo.html
+size: (434, 600)
+assert: (".top-doc", "open", "")
+click: (8, 280) // This the position of the top doc comment toggle
+assert-false: (".top-doc", "open", "")
+click: (8, 280)
+assert: (".top-doc", "open", "")
```

Comment:
This test seems both too fragile (depends on exact pixel offsets) and not specific enough (if the position is only off by a few pixels, the click might still succeed). Do we have access to the DOM in these tests? Can we check something like "the toggle doesn't overlap with the left side of the screen" or "the toggle's calculated left position is at least N pixels away from the left side of the screen?"

Reply:
The framework is not that smart, it doesn't allow computation. I can eventually add a click event just 1 pixel on the right of the toggle to ensure it's not there?

Reply:
> I can eventually add a click event just 1 pixel on the right of the toggle to ensure it's not there?

You mean 1 pixel to the right of the left-hand side of the screen? That seems fine, though still brittle.

In general there are a lot of things that differ in layout between desktop and mobile layouts. I don't think it's practical to write unittests for each of them. Is there a reason to write a unittest for this one? Well, maybe, since it's regressed once so far. But in general what would a test framework look like that verified certain layout properties held on mobile vs desktop? It seems like we want an entirely different paradigm for testing that. (idle thoughts; don't need to be fixed as part of this PR :-))

Reply:
The layout of rustdoc is supposed to not change much. By checking such small details, we can ensure that no unexpected changes ocurred (which happened way too many times ><). Let me add a click on the right and left of the toggle to ensure its position then.